docs/blog/newsletter-may-2017.rst

```diff
@@ -0,0 +1,118 @@
+.. post:: May 1, 2017
+   :tags: newsletter, 
+
+**************************************
+Write the Docs Newsletter - May 2017
+**************************************
+
+Write the Docs Portland is in Less than 2 Weeks!
+################################################
+
+`Write the Docs Portland <http://www.writethedocs.org/conf/na/2017/>`_ is almost here! If you're coming, we're looking forward to welcoming you. If you can't make it, maybe `Write the Docs Prague <http://www.writethedocs.org/conf/eu/2017/>`_ in September? Just a thought...
+
+The Portland conference is sold out again this year, but if you want to be on the waiting list in case of cancellations, please [**need to add this info**]
```

Comment:
https://ti.to/writethedocs/write-the-docs-na-2017/ waiting list on a ticket type basis. There are approx 10 people on there now. 

Please rewrite as needed